tools/wptrunner/wptrunner/testloader.py

```diff
@@ -383,11 +383,17 @@ def group_metadata(cls, state):
     def group(self):
         if not self.current_group or len(self.current_group) == 0:
             try:
-                self.current_group, self.current_metadata = self.test_queue.get(block=False)
+                self.current_group, self.current_metadata = self.test_queue.get(block=True, timeout=5)
             except Empty:
                 return None, None
```

Comment:
So it's always an error to get this exception now, correct? Could we log it at warning level so we have some chance ofnoticing such a bug?

Reply:
Previously the checks failed because `add_sentinal` should be a class method. After fixing that, now all the checks have passed.

Reply:
Thanks, and sorry I misread your previous comment. Updated...

Reply:
jpraham@, can you help review this? I need this for another task. Thanks!